azure-functions/acbs-function/src/functions/activity-get-facility-loan-id.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * This function is an Azure Durable sub-orchestrator function.
+ * This function cannot be invoked directly and is rather executed by an Azure durable orchestrator
+ * function.
+ *
+ * @module acbs-get-facility-loan-id
+ */
+
+/*
+ * GET Loan ID for facility DAF
+ * *****************************
+ * This DAF (Durable Activity Function) is never invoked directly.
+ * It is invoked via DOF (Durable Orchestrator Function).
+ *
+ * Pre-requisites
+ * --------------
+ * 0. 'npm install durable-functions'
+ * 1. Durable  HTTP trigger function (acbs-http)
+ * 2. Durable Orchestrator function (DOF) (acbs-amend-facility)
+ *
+ * ------------------
+ * HTTP -> DOF -> DAF
+ * ------------------
+ */
+const df = require('durable-functions');
+const api = require('../../api');
+const { isHttpErrorStatus } = require('../../helpers/http');
+
+df.app.activity('get-loan-id', {
```

Comment:
The name of the file doesn't match the DAF name

Reply:
Also the name `loan` suggests its not used for other facilities but I assume that's not the case?